### ISSUE TITLE
Send a reminder email 14 days before a renewal for any subscription >=180 days

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#7808]
In a VPN pricing meeting yesterday, the proposed plans for the upcoming European launch included a monthly plan, a six-month plan and an annual plan.

IIUC, we may need to send subscribers for these longer plans one or more additional emails due to the longer plan length. I didn't quite follow the logic, but I believe it was related to either autorenewal or expiration at the end of the period.

[~wclouser@mozilla.com], can you clarify if there are any additional email requirements for these new plan lengths in your next meeting with the Legal team?

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-3166)
┆Issue Number: FXA-3166

**Comment by data-sync-user** [comment]
➤ Wil Clouser commented:

I will ask.  Thanks for bringing it up.  The next meeting isn't until the 22nd.

**Comment by data-sync-user** [comment]
➤ Bianca Danforth commented:

Thanks [~wclouser@mozilla.com] . In that case, given our April launch timeline and the possible additional work involved, do you think it's worth asking them in advance of the next meeting on 3/22 (and is that an option)?

**Comment by data-sync-user** [comment]
➤ Wil Clouser commented:

I think it's ok to stick with our current approved plan -- emails are sent when subscriptions are renewed/billed.  I haven't had any other signals that we'd need to change that.   But maybe something specific happened to file this bug?

**Comment by data-sync-user** [comment]
➤ Bianca Danforth commented:

Well, there are a couple of firsts for the April launch that may change the legal requirements (and that seemed to be the suggestion in the meeting) – this is the first time subscriptions will be offered:

  * in Europe (different consumer protection laws)
  * for plan lengths > 1 month

Anyway, it's possible no changes are necessary; I just wanted to be sure.

**Comment by data-sync-user** [comment]
➤ Wil Clouser commented:

Hi Bianca.  Your instincts were correct, thanks for filing.  Legal has asked us to send a reminder email 14 days before a renewal for any subscription >=180 days.  The reminder email needs to tell the user when the next bill will be and how much they will pay.  Other than that, there is no specific wording (although I suspect continuing to link to our unsubscribe option would be appreciated).

So, we should implement the email logic, and we'll need email copy, and also we'll have to think about a way that QA can test this without waiting 6 months..

**Comment by data-sync-user** [comment]
➤ Bianca Danforth commented:

CCing [~lnorton@mozilla.com] and [~jecook@mozilla.com] for visibility. Maybe we can discuss this ticket in the next design check-in?

**Comment by data-sync-user** [comment]
➤ Bianca Danforth commented:

Also thank you Wil for checking quickly!

**Comment by data-sync-user** [comment]
➤ Bianca Danforth commented:

We plan to use an existing template for this e-mail.

[~byee@mozilla.com], we had discussed in our weekly PayPal meeting that you would take the action item to provide the copy for this new e-mail. Can you share it when that's ready? This ticket is blocked on the copy.

**Comment by data-sync-user** [comment]
➤ Brian Yee commented:

Sent this over to legal and marketing to review.  https://docs.google.com/document/d/1PbxlKSVt41kpU63axX6D8w0L6gnUyn1kmYzAkz0zf1U/edit?usp=sharing ( https://docs.google.com/document/d/1PbxlKSVt41kpU63axX6D8w0L6gnUyn1kmYzAkz0zf1U/edit?usp=sharing )

**Comment by data-sync-user** [comment]
➤ Brian Yee commented:

The link above has been approved by legal.  Please check the link to check payment method, as I don't have an account that has a payment method saved to it.  Thx
